src/main/java/uk/gov/hmcts/probate/model/DocumentType.java

```diff
@@ -12,7 +12,7 @@
     LEGAL_STATEMENT_PROBATE("legalStatementProbate"),
 
     @JsonProperty("legalStatement")
-    LEGAL_STATEMENT_PROBATE_TRUST_CORPS("legalStatementGrantOfProbate"),
+    LEGAL_STATEMENT_PROBATE_TRUST_CORPS("legalStatementProbate"),
```

Comment:
I think we need to be careful here. Would be good to check with a senior dev as there are currently 2 types of legal statement. 1 before TC, so old schema (legalStatementProbate), and one after (legalStatementGrantOfProbate). They behave differently so don't think it's a good idea to change it.

Reply:
i'll revert this, I think that is right, and will pass on to business

Reply:
especially to the same name